components/badge/badge.css

```diff
@@ -0,0 +1,5 @@
+@import url('../../starterFiles/abstracts/variables.css');
+@import url('../../starterFiles/base/reset.css');
+@import url('../../starterFiles//utils/util.css');
```

Comment:
The double slash ( // ) in the import url is by mistake, I guess. Remove it. Also for line 4 and 5

Reply:
no it's the path for the files.